spec/system/consumer/caching/shops_caching_spec.rb

```diff
@@ -29,7 +29,7 @@
 
     it "keeps data cached for a short time on subsequent requests" do
       # Ensure sufficient time for requests to load and timed caches to expire
-      Timecop.travel(10.minutes.ago) do
+      travel(-10.minutes) do
```

Comment:
I'm curious about the reason for the change to `-10.minutes` here. 

I'm quite used to using the time helpers now, so at first glance it seems uglier. (but I have to admit it took a while to get used to them, so maybe this way is better for people new to Rails?)

Reply:
Yes, it is a bit ugly. And the minus is difficult to see. I applied a simple rule to use `travel` for all distances and `travel_to` to all fixed times. But I'm not sure what's better:

```rb
travel(-10.minutes) # or
travel_to(10.minutes.ago)
```

The first is shorter and the second _maybe_ easier to read.

Reply:
Thanks for explaining. I think I would currently choose the latter, for its natural language flow, but honestly it's personal opinion which may change in the future. I usually prefer shorter code anyway. 🤷  Both are good.